ckan/migration/revision_legacy_code.py

```diff
@@ -68,6 +68,9 @@ def package_dictize_with_revisions(pkg, context):
         res = model.resource_table
     else:
         res = revision_model.resource_revision_table
+    mm_col = res._columns.get('metadata_modified')
+    if mm_col is not None:
+        res._columns.remove(mm_col)
```

Comment:
probably good to add a comment here to explain why we need to do this

Reply:
Done